.travis.yml

```diff
@@ -14,4 +13,3 @@ before_script:
 addons:
   hosts:
     - dynamodb-local
-script: "MIX_ENV=test mix local.hex --force && MIX_ENV=test mix do deps.get, test"
```

Comment:
This is a genuine question, I wasn't saying you had to add it back. I'm trying to figure out why you removed it initially, and why you've now added it back.